static/lang/en.json

```
@@ -3195,6 +3196,8 @@
         "SpellTypeSave": "Saving Throw",
         "SpellTypeUtility": "Utility",
         "SpellCategorySpell": "Spell",
+        "SpellCategoryCantripSpell": "Cantrip Spell",
```

```suggestion
        "SpellCategoryCantripSpell": "Cantrip Spell",
```
Maybe "Cantrip" is fine to use here so we could just use `SpellCantripLabel` instead.